spec/models/collection_spec.rb

```diff
@@ -397,7 +397,7 @@
     end
   end
   describe "catalog-rest atom result to open search conversion" do
-    it "should convert a single catalog-rest result to a single open search result" do
+    it "should convert a single catalog-rest fedeo collection result to a single open search result" do
```

Comment:
Shouldn't we be adding a test rather than changing one?

Reply:
The reason for altering this test rather than adding another is that the very next unit test in the file was created by copy-pasting the one changed here - the same test case is repeated twice. So rather than repeat the same test case for essentially the third time, I opted to change this one.

Reply:
OK. If we really do exactly the same test then fair enough.